Create Outreach SIG past programs page (#5013)
jenkins-infra/jenkins.io: content/sigs/advocacy-and-outreach/outreach-programs/outreach-programs-history.adoc

---
layout: project
title: "Outreach Past Programs"
section: sigs
sigId: "advocacy-and-outreach"
tags:
  - advocacy-and-outreach
  - advocacy-and-outreach-sig
  - outreach-programs
  - community
  - outreach
  - programs
  - hacktoberfest
opengraph:
  image: /images/gsoc/jenkins-gsoc-logo_small.png
---

> This page lists the community programs we participated to during the past years.

Reviewer: ```suggestion
This page lists the community programs we've supported in the past.
```
